CONTRIBUTING.md

```diff
@@ -37,7 +37,8 @@ To do so, you only need to follow these steps:
   1. Check if you already have existing GPG keys in your local computer following [this guide](https://help.github.com/articles/checking-for-existing-gpg-keys/).
   2. If you don't have any, generate a new GPG key following [this guide](https://help.github.com/articles/generating-a-new-gpg-key/).
   3. Add the GPG key to your GitHub account following [this guide](https://help.github.com/articles/adding-a-new-gpg-key-to-your-github-account/).
-  4. Sign your commits with the GPG key following [this guide](https://help.github.com/articles/signing-commits/).
+  4. Tell Git about your signing key following [this guide](https://help.github.com/articles/telling-git-about-your-signing-key/).
```

Comment:
I was thinking about just replacing this whole numbered list with this link: https://help.github.com/articles/managing-commit-signature-verification/

It already has the links to every specific step, and is also ordered appropriately.

Reply:
Numbered list are more straightforward and less obscure from a readme perspective, my 0.02€!